plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -507,6 +513,12 @@ static struct flb_config_map config_map[] = {
      "the log message will be sent to CloudWatch."
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "extra_user_agent", NULL,
+     0, FLB_FALSE, 0,
+     "Append the user agent with this string to track usage."
```

Comment:
Change to: 

"This option appends a string to the default user agent. AWS asks that you not manually set this field yourself, it is reserved for use in our vended configurations, for example, EKS Container Insights."